### PR TITLE
webui: Dont run browser as root

### PR DESCRIPTION
**Pull request by adamkankovsky**
Inspirated by: https://github.com/rhinstaller/anaconda/pull/5058

This changes for liveOS and boot.iso was tested and works

### CODE REVIEW

File: dockerfile/anaconda-iso-creator/pkexec-command.patch [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+From: Adam Kankovsky <adamkankovsky@gmail.com>
+Date: Thu, 3 Oct 2024 09:52:26 +0200
+Subject: [PATCH] Update cleanup for pkexec command
+
+---
+ share/templates.d/99-generic/runtime-cleanup.tmpl     | 1 -
+ share/templates.d/99-generic/runtime-postinstall.tmpl | 4 ++++
+ 2 files changed, 4 insertions(+), 1 deletion(-)
+
+diff --git a/share/templates.d/99-generic/runtime-cleanup.tmpl b/share/templates.d/99-generic/runtime-cleanup.tmpl
```
Comment (KKoukiou) [inline dockerfile/anaconda-iso-creator/pkexec-command.patch:11]: This needs to be fixed upstream Please. 

Create a PR and add a link here with FIXME to remove: https://github.com/weldr/lorax/blob/master/share/templates.d/99-generic/runtime-cleanup.tmpl

**Review by KKoukiou — CHANGES_REQUESTED** [review]

**Comment by jkonecny12** [comment]
This PR conflicts with https://github.com/rhinstaller/anaconda/pull/5946

**Comment by jkonecny12** [comment]
But I would like to see both merged soon.

**Pull request merged**